manifests/agent.pp

```diff
@@ -18,6 +18,10 @@
   $pxp_ping_interval     = undef,
 ) {
 
+  File {
+    backup => false
+  }
+
```

Comment:
This should only disable backup while applying the `clamps::agent` class, which isn't a part of performance testing. It's currently helping because every time the `clamps::agent` class is applied it overwrites every script with new facts. I'm going to evaluate to having the mock script use facter, or write a single facts file that all the scripts load.